### PR TITLE
Accounts Styling 

### PR DESCRIPTION
**Pull request by bzw24**
https://trello.com/c/9NqxUGlT

### Accounts Page TailwindCSS Styling Updates
- Significantly improve responsiveness of all accounts components
- Deleted overflow on all forms, since they were cutting off the input indicators
- Updated handling of phone number input
- Changed all 'My Parking' to 'Reservations'
- Conditionally render menu based on role
- Slight padding change on Accounts Page

### CODE REVIEW

File: src/components/account/AccountSetting.jsx [hunk 1]
```diff
@@ -12,30 +12,17 @@ const AccountSetting = () => {
...
   const formattedPhoneNumber = useMemo(() => {
     return formatPhoneNumber(userData.phone_number);
-  }, [userData])
+  }, [userData]);
 
-  const handlePhoneNumberChange =
-    (e) => {
-      dispatch(setUserDataPhoneNumber(formatPhoneNumber(e.target.value)));
-    }
+  const handlePhoneNumberChange = (e) => {
```
Comment (clee4037) [inline src/components/account/AccountSetting.jsx:20]: Separation of concerns -- good job

**Review by clee4037 — APPROVED** [review]
Approve styling changes!

**Review by kjmak14 — APPROVED** [review]
Styling changes look good

**Review by justcord10 — APPROVED** [review]
Looks good, dont notice anything wrong

**Pull request merged**